### PR TITLE
Update 3.1.3xx branding

### PR DESCRIPTION
**Pull request by sfoslund**
- Internal merge
- Removing internal feeds
- Updating branding

### CODE REVIEW

**Review by wli3 — APPROVED** [review]

**Comment by sfoslund** [comment]
> Failed to retrieve information about 'Microsoft.DotNet.Arcade.Sdk' from remote source 'https://pkgs.dev.azure.com/dnceng/_packaging/dotnet3.1-internal-transport/nuget/v2/FindPackagesById()?id='Microsoft.DotNet.Arcade.Sdk'&semVerLevel=2.0.0'.

@wtgodbe do you recognize this error?

**Comment by wtgodbe** [comment]
Looks like the conditions for `setup private feed` credentials are wrong, it's trying to use the private feeds in the public build. I'll take a look

**Comment by wtgodbe** [comment]
@sfoslund where are you seeing that error?

To get around the HostModel test failures, you can update global.json to use the `3.1.104` or `3.1.202` SDKs

**Comment by sfoslund** [comment]
@wtgodbe I'm still seeing the arcade error after updating stage 0 here and with all the other linux build legs: https://dev.azure.com/dnceng/public/_build/results?buildId=645487&view=logs&j=fd8df696-9db3-57b4-fb8e-b67f5ba463c3&t=beec83df-ea61-5405-c7f4-1fb1e5d3044c&l=37

File: eng/build.yml [hunk 1]
```diff
@@ -188,6 +259,13 @@ jobs:
             continueOnError: true
 
     - ${{ if notIn(parameters.agentOs, 'Windows_NT', 'Windows_NT_FullFramework', 'Windows_NT_TestAsTools', 'Windows_Perf_CI', 'Linux_Perf_CI', 'Windows_Perf_Helix', 'Windows_Perf_CI_FullFramework', 'Windows_Perf_Helix_Fullframework', 'Linux_Perf_Helix') }}:
+      - task: Bash@3
```
Comment (wtgodbe) [inline eng/build.yml:262]: Here's the problem, this needs to be conditional. Same w/ line 241. I'll push a fix

**Comment by wtgodbe** [comment]
We have more:

> Expected command to pass but it did not.\r\nFile Name: F:\workspace\_work\1\s\.dotnet\dotnet.exe\r\nArguments: msbuild /t:Build F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj /restore /p:RuntimeIdentifier=win10-x64 /p:TestRuntimeIdentifier=win10-x64\r\nExit Code: 1\r\nStdOut:\r\nMicrosoft (R) Build Engine version 16.5.0+d4cbfca49 for .NET Core\r\nCopyright (C) Microsoft Corporation. All rights reserved.\r\n Restore completed in 52.04 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithRids\LibraryWithRids.csproj.\r\n Restore completed in 51.95 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithoutRid\LibraryWithoutRid.csproj.\r\n Restore completed in 51.96 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj.\r\n Restore completed in 51.95 ms for F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\LibraryWithRid\LibraryWithRid.csproj.\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: The "ResolvePackageAssets" task failed unexpectedly. [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: System.IO.FileNotFoundException: Could not load file or assembly 'NuGet.ProjectModel, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35'. The system cannot find the file specified. [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: File name: 'NuGet.ProjectModel, Version=5.6.0.5, Culture=neutral, PublicKeyToken=31bf3856ad364e35' [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.CacheReader.CreateReaderFromMemory(ResolvePackageAssets task, Byte[] settingsHash) [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.CacheReader..ctor(ResolvePackageAssets task) in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 477 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.ReadItemGroups() in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 301 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.Tasks.ResolvePackageAssets.ExecuteCore() in //src/Tasks/Microsoft.NET.Build.Tasks/ResolvePackageAssets.cs:line 294 [F:\workspace\_work\1\s\artifacts\tmp\Release\It_builds_a_R---7CC012B9\App\App.csproj]\r\nF:\workspace\_work\1\s\artifacts\bin\Release\Sdks\Microsoft.NET.Sdk\targets\Microsoft.PackageDependencyResolution.targets(234,5): error MSB4018: at Microsoft.NET.Build.T

**Comment by wtgodbe** [comment]
Is this the nuget.client Stage0? I don't know why that wouldn't be resolved by updating the SDK. Who's our contact on the nuget team?

**Comment by wli3** [comment]
Error looks like need to update stage 0

**Comment by wtgodbe** [comment]
Ahhh, actually I think it's the inverse of stage0 - we updated to the 3.1.202 to solve the HostModel stage0, but in doing so brought in an SDK that has a nuget.client that this branch doesn't know about (whatever nuget.client was inserted into 3.1.202). I think we should use the 3.1.300 SDK instead - it's not shipped yet, but we can use it by referencing its nonstable version. Will push an update.

**Comment by wli3** [comment]
Don't see this is anything special than normal nuget dlls. We need to wait for dotnet/installer's counter part to have a build. And insert it here.

**Comment by wtgodbe** [comment]
@wli3 there's no nuget.client update in this PR, though - I thought that was only necessary when we do nuget insertions

**Comment by wli3** [comment]
should be blocked by this https://github.com/dotnet/installer/pull/7480

**Comment by wli3** [comment]
maybe you are right. Let's see

**Comment by wtgodbe** [comment]
Ah, crap, the 3.1.300 SDK isn't anywhere public yet, even with the non-stable version. So I think you're right @wli3, we need the stage0 from https://github.com/dotnet/installer/pull/7480, which will have the right nuget.client & HostModel

**Comment by wli3** [comment]
@wtgodbe the failure of this is also understood?

**Comment by wtgodbe** [comment]
Yes, we need a stage0, which is currently building: https://dev.azure.com/dnceng/internal/_build/results?buildId=647154&view=results

**Pull request merged**